lib/crawly/bench/bench_router.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Crawly.Bench.BenchRouter do
```

Comment:
I wanted to check if it's possible (or if we should do it) to abstract it to a separate node? E.g. having an exs file, which is started separately from Crawly. So potentially it's possible to run it as a standalone process, to avoid any possible collision.